src/i18n/en/common.json

```diff
@@ -315,6 +315,23 @@
       "close": "Close"
     }
   },
+  "header": {
+    "actions": {
+      "favourites": {
+        "title": "Favourites"
+      },
+      "userManagement": {
+        "logIn": "Log in",
+        "logOut": "Log out"
```

Comment:
Good change, I hadn't noticed we used "sign in/out" only in one place and "log in/out" everywhere else.